tests/Localization/SoDjTest.php

```diff
@@ -17,210 +17,210 @@ class SoDjTest extends LocalizationTestCase
 
     const CASES = [
         // Carbon::parse('2018-01-04 00:00:00')->addDays(1)->calendar(Carbon::parse('2018-01-04 00:00:00'))
-        'Tomorrow at 12:00 subaxnimo',
+        'Tomorrow at 12:00 AM',
```

Comment:
Tommorow is 'Beri' in somali.
any AM is "GH"
any PM is 'GD'

Reply:
I guess "at" should be translated too. Here is the default (English) translations:
```php
    'calendar' => [
        'sameDay' => '[Today at] LT',
        'nextDay' => '[Tomorrow at] LT',
        'nextWeek' => 'dddd [at] LT',
        'lastDay' => '[Yesterday at] LT',
        'lastWeek' => '[Last] dddd [at] LT',
        'sameElse' => 'L',
    ],
```
Could you please add and translate it in https://github.com/Abdifatahz/Carbon/blob/patch-1/src/Carbon/Lang/so.php

I can't find "GH" / "GD" anywhere, all the reference I found use AM or subaxnimo. Can you show some official sources?

Reply:
Added Calendar.
Thank you @kylekatarnls .

Reply:
You can skip GH and GD it's just shorthand for GelinHore < 12PM and gelinDanbe >= 12PM.

Reply:
OK